package Prova.Back.Exception;

public class AlreadyExistsException {

}
